etl/steps/data/garden/hmd/2022-11-04/life_tables.py

```diff
@@ -0,0 +1,97 @@
+import json
+from typing import List, cast
+
+import pandas as pd
+from owid.catalog import Dataset, Table
+from owid.catalog.utils import underscore_table
+from owid.datautils import geo
+from structlog import get_logger
+
+from etl.helpers import Names
+from etl.paths import DATA_DIR
+
+log = get_logger()
+
+# naming conventions
+N = Names(__file__)
+
+
+def run(dest_dir: str) -> None:
+    log.info("life_tables.start")
+
+    # read dataset from meadow
+    ds_meadow = Dataset(DATA_DIR / "meadow/hmd/2022-11-04/life_tables")
```

Comment:
For the future: Something I usually do is to define versions at the top of the script, so that, on the next update, it's clear what needs to be manually changed. In this case, it would be `MEADOW_VERSION="2022-11-04`.